src/backend/loader.js

```diff
@@ -14,4 +16,54 @@ os.homedir = () => nodejsProjectDir
 process.cwd = () => nodejsProjectDir
 process.env = process.env || {}
 
-import './index.js'
+const { values } = parseArgs({
+  options: {
+    sentryEnvironment: { type: 'string' },
+    sentryUserId: { type: 'string' },
+    metricsIsEnabled: { type: 'string' },
+  },
+  strict: false,
+})
+
+const { sentryEnvironment, sentryUserId, metricsIsEnabled } = values
+
+if (typeof metricsIsEnabled !== 'string')
+  throw new Error('backend did not receive metricsIsEnabled')
+if (typeof sentryUserId !== 'string')
+  throw new Error('backend did not receive sentryUserId')
+if (typeof sentryEnvironment !== 'string')
+  throw new Error('backend did not receive sentryEnvironment')
+
+const sentryDebug = sentryEnvironment === 'development'
+const initialScope = sentryUserId ? { user: { id: sentryUserId } } : undefined
+
+const logLevels = ['error']
```

Comment:
This removes the need for @ts-expect-error below - it ensures that these levels are typed as string literals, rather than just a generic string.

```suggestion
/** @type {Array<"error" | "log" | "warn">} */
const logLevels = ['error']
```